src/components/markdown/markdown-parser.ts

```diff
@@ -67,6 +70,7 @@ function getWhiteList(allowedComponents: CustomElement[]): Schema {
             ...defaultSchema.attributes,
             p: [['className', 'MsoNormal']], // Allow the class 'MsoNormal' on <p> elements
             '*': ['style', 'width'], // Allow `style` and 'width' attribute on all elements
+            a: ['href', 'target', 'rel'], // Allow `target`, `rel` and `href` on <a> elements
```

Comment:
Looks good, maybe good if we make it alphabetically ordered. But it doesn't block me approving though
```suggestion
            a: ['href', 'rel', 'target'],, // Allow `target`, `rel` and `href` on <a> elements
```